InitBuilder.js

```diff
@@ -0,0 +1,79 @@
+
+/*
+new IntBuilder(int) // constructor takes starting integer, if not passed starts with 0;
+.plus(...n)         // take infinite number of integers and sum all with stored value;
+.minus(...n)        // take infinite number of integers and subtract from stored value; 
+.multiply(n)        // multiply param n on stored value;
+.divide(n)          // leaves integer part of division stored value on n; 
+.mod(n)             // leaves remainder of the division stored value with on n;
+.get()              // returns stored value;
+*/
+"use strict"
+
+class  baseBilder{  // 
+    constructor(value){
+        this.value = value 
+    }
+    plus(...n) {
+        this.value =  this.value  + n.reduce((sum, value) => sum + value) // this
+        return this
+    }
+    divide(n){
+        this.value = this.value / n
```

Comment:
please pay attention to syntax, you are missing `;` here and in other places. In general it's a good idea to configure `eslint` for such purpose